cognite/client/_api/transformation_schema.py

```diff
@@ -0,0 +1,31 @@
+from cognite.client import utils
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes import TransformationDestination, TransformationSchemaColumnList
+
+
+class TransformationSchemaAPI(APIClient):
```

Comment:
just one method? how is schema created?

Reply:
it is a read-only endpoint, it tells the user the schema that a sql select should have to be able to write a given resource type

Reply:
schema is not a resource type, it is a utility endpoint that returns the schema for a resource type.